shumai/tensor/tensor.ts

```diff
@@ -294,89 +299,104 @@ export class Tensor {
       this.stats = obj.stats
       this.grad = obj.grad
       this.op = obj.op
+      if (is_tidy_run) this.track_tidy_ptr()
       return
     }
     if (obj.hasOwnProperty('_ptr')) {
       this._injest_ptr(obj._ptr)
+      if (is_tidy_run) this.track_tidy_ptr(obj._ptr)
```

Comment:
`obj._ptr` should be the same as `this.ptr` at this line, right?

Reply:
I think it'd be best to simply use `this.ptr` for now and then we can optimize the getter later (since we can always cache `this._ptr = ptr(this.underlying)`)